test/integration/admin/api/account/proxy_configs_controller_test.rb

```diff
@@ -111,6 +111,24 @@ def setup
     assert_same_elements expected_proxy_config_ids, response_proxy_config_ids
   end
 
+  test '#index of latest verisons filtered by host is only found among those' do
+    services = FactoryBot.create_list(:simple_service, 2, :with_default_backend_api, account: provider)
+    services.each do |service|
+      hosts = %w[3scale.localhost example.com]
```

Comment:
We can move this out of the loop:
```suggestion
      hosts = %w[3scale.localhost example.com]
```